src/app/components/context-menu/context-menu.directive.ts

```diff
@@ -69,6 +61,7 @@ export class ContextActionsDirective {
   ) {}
 
   private execute(event: MouseEvent) {
+    // todo: review this in ADF
```

Comment:
can you create a ticket to review the ADF part?

Reply:
created https://issues.alfresco.com/jira/browse/ADF-3679